docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -48,25 +48,25 @@ Set the systemd unit name prefix for containers. The default is *container*.
 
 Set environment variables to the systemd unit files.
 
-If an environment variable is specified without a value, Podman will check the host environment for a value and set the variable only if it is set on the host. As a special case, if an environment variable ending in __*__ is specified without a value, Podman will search the host environment for variables starting with the prefix and will add those variables to the systemd unit files.
+If an environment variable is specified without a value, Podman checks the host environment for a value and set the variable only if it is set on the host. As a special case, if an environment variable ending in __*__ is specified without a value, Podman searches the host environment for variables starting with the prefix and adds those variables to the systemd unit files.
 
 #### **--files**, **-f**
 
-Generate files instead of printing to stdout.  The generated files are named {container,pod}-{ID,name}.service and will be placed in the current working directory.
+Generate files instead of printing to stdout.  The generated files are named {container,pod}-{ID,name}.service and are placed in the current working directory.
 
-Note: On a system with SELinux enabled, the generated files will inherit contexts from the current working directory. Depending on the SELinux setup, changes to the generated files using `restorecon`, `chcon`, or `semanage` may be required to allow systemd to access these files. Alternatively, use the `-Z` option when running `mv` or `cp`.
+Note: On a system with SELinux enabled, the generated files inherits contexts from the current working directory. Depending on the SELinux setup, changes to the generated files using `restorecon`, `chcon`, or `semanage` may be required to allow systemd to access these files. Alternatively, use the `-Z` option when running `mv` or `cp`.
 
 #### **--format**=*format*
 
-Print the created units in specified format (json). If `--files` is specified the paths to the created files will be printed instead of the unit content.
+Print the created units in specified format (json). If `--files` is specified the paths to the created files is printed instead of the unit content.
```

Comment:
```suggestion
Print the created units in the specified format (json). If `--files` is specified, the paths to the created files are printed instead of the unit content.
```